features/step_definitions/purap/requisition.rb

```diff
@@ -42,7 +59,7 @@
     on CommodityCodeLookupPage do |comm_page|
       comm_page.sensitive_data.select_value(@item.determine_commodity_code_of_type(commodity_code_type))
       comm_page.search
-      comm_page.wait_for_search_results
+      comm_page.wait_for_search_results(45)
       comm_page.return_random
     end
```

Comment:
While verifying the other suggested changes, I have discovered that increasing this wait time did not fix the issue that occurs during the first run cache building.  Alternate code suggestion being made below:
comm_page.search
      comm_page.wait_for_search_results
      begin
        comm_page.return_random
      # first time cache building causes this, wait a bit longer and try again
      rescue Watir::Exception::UnknownObjectException
        puts "Watir::Exception::UnknownObjectException rescued for Commmodity Code search. Waiting a bit longer for search results before attempting return_random a second time."
        sleep(15)
        comm_page.return_random
      end